_data-prepper/core-apis.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Core APIs
+nav_order: 2
+---
+
+# Core APIs
+
+All Data Prepper instances expose a server with some control APIs. By default, this server runs on port 4900. Some plugins, especially source plugins may expose other servers. These will be on different ports, and their configurations are independent of the core API. For example, to shut down Data Prepper, you can run the following curl request:
+
+```
```

Comment:
I think you can use a specific code format for cURL: ```curl
```curl
curl -X POST...
```

Reply:
@cwillum Can you please clarify what you mean? I'm not seeing a specific code format for this.

Reply:
Nothing pressing. I'm talking about the way markdown represents the code block. For example, we often use ...
~~~
```json
json content
```
~~~
...to put code in that format. You can also do it for YAML files:

~~~
```yaml
yaml content
```
~~~

I thought you should be able to do the same for a cURL command. But I can't find a reference. Probably OK to overlook this comment and move on.